app/controllers/coronavirus_form/expert_advice_type_controller.rb

```diff
@@ -38,11 +38,11 @@ def submit
 
   PAGE = "expert_advice_type"
   NEXT_PAGE = "offer_care"
+  ALLOWED_VALUES = I18n.t("coronavirus_form.questions.#{PAGE}.options").map { |_, item| item.dig(:label) }
```

Comment:
what would stop you memoizing this in a similar way to what you did with [reference number](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/145/files#diff-ebeb2548534e3e5e7c407258f3d531bdR27), even if it's fairly lightweight, a map method has some overhead right?

Reply:
Since this is a constant `I18n.t("coronavirus_form...` will only get evaluated once when a new instance is created, I think.